### PR TITLE
Adds release note for Intel E810-XXVDA4T

### PR DESCRIPTION
**Pull request by stevsmit**
Version(s):
4.13 

Issue:
https://issues.redhat.com/browse/OSDOCS-6094

Link to docs preview:

SME approved.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -720,6 +720,15 @@ You can now migrate a cluster that runs on {rh-openstack} and uses Kuryr to OVN-
 
 For clusters that run on {rh-openstack} and use OVN-Kubernetes, manually reassigning floating IP addresses for reservation ports is no longer necessary. If a reservation port is removed from one node and recreated on another one, the reassignment now happens automatically.
 
+[id="ocp-4-13-networking-supported-hardware-for-sr-iov"]
+==== Supported hardware for SR-IOV (Single Root I/O Virtualization)
+
+{product-title} {product-version} adds support for the following SR-IOV devices: 
+
+* E810-XXVDA4
```
[the remark this answers is not in the document]
Comment (stevsmit) [reply release_notes/ocp-4-13-release-notes.adoc]: Fixed. Sorry about that.

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -2011,6 +2020,11 @@ In the following tables, features are marked with the following statuses:
 |Not Available
 |Technology Preview
 
+|Intel E810-XXVDA4T
```
Comment (wizhaoredhat) [inline release_notes/ocp-4-13-release-notes.adoc:2023]: It does seem like not all the NICs that we have supported are in this table. E.g.
```
|Broadcom
|BCM57414
|14e4
|16d7
|Broadcom
|BCM57508
|14e4
|1750
|Intel
|X710
|8086
|1572
|Intel
|XL710
|8086
|1583
|Intel
|XXV710
|8086
|158b
|Intel
|E810-CQDA2
|8086
|1592
|Intel
|E810-2CQDA2
|8086
|1592
|Intel
|E810-XXVDA2
|8086
|159b
|Intel
|E810-XXVDA4
|8086
|1593
```

Comment (stevsmit) [reply release_notes/ocp-4-13-release-notes.adoc:2023]: @wizhaoredhat That is likely. We don't add every single device to this table. As a Technology Preview table, we keep entries in for 3 releases (for example, for 4.13 the table is for 4.11, 4.12, and 4.13). Unless the devices you listed above were added in one of those releases, that specific table will not list them. 

We provide the full list of devices here: https://docs.openshift.com/container-platform/4.12/networking/hardware_networks/about-sriov.html#supported-devices_about-sriov 

**If any of these devices were added in 4.13, 4.12, or 4.11, and we missed them, I can add them to this list.**

Comment (wizhaoredhat) [reply release_notes/ocp-4-13-release-notes.adoc:2023]: Ah I see. Thanks for clarifying!

**Comment by wizhaoredhat** [comment]
I would change the PR title and commit title to be E810-XXVDA4T instead of E810-XXVDA4.

**Comment by vrindle** [comment]
LGTM

**Review by kalexand-rh — APPROVED** [review]
LGTM. The local build looks good, and the content at the link in line 730 will be added on https://github.com/openshift/openshift-docs/pull/60057.

**Pull request merged**